docs/r2/tensorboard_scalars_and_keras_basic.ipynb

```diff
@@ -0,0 +1,452 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "tensorboard_scalars_and_keras_basic.ipynb",
```

Comment:
Is this still "basic" now that custom scalars are included? Would there be a more advanced tutorial?

Reply:
Renamed.